sw/device/tests/rstmgr_alert_info_test.c

```diff
@@ -115,9 +115,11 @@ typedef enum test_round {
   kRoundTotal = 3
 } test_round_t;
 
+#define ALERT_CAUSE_CNT ((ALERT_HANDLER_PARAM_N_ALERTS - 1) / 32) + 1
```

Comment:
If you want this to round up correctly, you could do `(ALERT_HANDLER_PARAM_N_ALERTS + 31) / 32`